source/scss/components/_form-checkbox.scss

```diff
@@ -0,0 +1,52 @@
+.checkmark {
+  border: solid 1px $secondary;
```

Comment:
Do not use the variable secondary, since it is blue and not gray. Use directly $gray-600

Reply:
The intention is to use $secondary